docs/tutorials/dotnot.md

```diff
@@ -0,0 +1,420 @@
+---
+layout: default
+title: "Migrating and deploying .NET apps (.Net framework version 4.x) to Kubernetes"
+permalink: /tutorials/dotnet/
+parent: Tutorials
+nav_order: 7
+---
+
+# Migrating and deploying .NET applications (4.x .NET Framework) to Kubernetes
+
+## Description
+
+In this tutorial, we will learn how to migrate and deploy .NET applications developed for 4.x versions of .NET framework, to Kubernetes cluster using Move2Kube. Here, we are going to use the sample [WCF](https://docs.microsoft.com/en-us/dotnet/framework/wcf/whats-wcf) service from [samples/wcfservice](https://github.com/konveyor/move2kube-demos/tree/main/samples/wcfservice).
```

Comment:
Would be good to call out "using windows containers" somewhere. Otherwise it might give the impression that it will more to dotnet core and create linux containers.

Reply:
done